eventbridge-pipes-amazonmq-to-eventbridge-sam/eventbridge-pipes-amazonmq-to-eventbridge-sam-pattern.json

```diff
@@ -0,0 +1,60 @@
+{
+  "title": "Amazon MQ broker as a producer for a Amazon EventBridge Message Bus using Amazon EventBridge Pipes",
+  "description": "This pattern showcases the use of Amazon EventBridge Pipes to forward events to an Amazon EventBridge custom Message Bus. Using this pattern, events placed on a queue on an Amazon MQ (ActiveMQ) broker are consumed by Amazon EventBridge Pipes which performs a basic transformation and places events on a Amazon EventBridge custom Message Bus. A rule defined on the Amazon EventBridge custom Message Bus writes events received to an Amazon CloudWatch Log Group.",
+  "language": "YAML",
+  "level": "200",
+  "framework": "Serverless Framework",
```

Comment:
Should this be SAM?

Reply:
The Framework is SAM however the Language I believe should be YAML based on the convention (see https://serverlessland.com/patterns?framework=SAM&language=YAML), please let me know if this is incorrect

Reply:
Yes the framework should be `SAM` instead of `Serverless Framework`

Reply:
🤦‍♂️ Ah I understand now, all clear. I have corrected the framework attribute in ecac078310f7653c13cae684f4876ec016faa96a